core/lib/spree/core/mail_method.rb

```diff
@@ -4,9 +4,9 @@ class MailMethod
       def initialize(options={})
       end
 
-      def deliver!(mail)
+      def deliver_now!(mail)
         if Config.enable_mail_delivery
-          mailer.deliver_now!(mail)
+          mailer.deliver!(mail)
```

Comment:
I futz'd with this quite a bit. It's a spree method call not ActionMailer. Goal was not to change spree call signatures.